src/fizzBuzz.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+export default function fizzBuzz() {
+  var arrayNums = [];
+  for (var i = 1; i < 101; i++){
+    arrayNums.push(i)
+  }
+
+  for(var i = 1; i < arrayNums.length; i++) {
+    if (arrayNums[i] % 3 == 0 ){
+      arrayNums.splice(i, 1, "Fizz")
+  }
+    if (arrayNums[i] % 5 == 0){
+    arrayNums.splice(i, 1, "Buzz")
+  }
+    if (i % 15 == 0 ){
+    arrayNums.splice(i -1 , 1, "FizzBuzz")
+  }
+  }
+  return arrayNums;
+
+
+  }
```

Comment:
you can do this with one loop instead of two
```js

function fizzBuzz(){
  const numbers = [];
  
  for (let index = 0; index <= 100; index++){
    let number = index + 1
    if (number % 3 == 0 && number % 5 == 0){
      numbers[index] = "FizzBuzz"
    }else if (number % 3 == 0){
      numbers[index] = "Fizz"
    }else if (number % 5 == 0){
      numbers[index] = "Buzz"
    }else{
      numbers[index] = number
    }
  }

  return numbers;
}
```